package.json

```diff
@@ -30,39 +30,39 @@
   },
   "homepage": "https://github.com/transitmatters/new-train-tracker#readme",
   "dependencies": {
-    "@types/react": "^18.0.1",
-    "@types/react-dom": "^18.0.0",
     "bezier-js": "^2.5.1",
     "classnames": "^2.2.6",
-    "react": "^18.0.0",
-    "react-dom": "^18.0.0",
-    "react-favicon": "1.0.0",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
```

Comment:
Can we just use React 17 for now? I don't think we are using any features from React 18

Reply:
Yeah. That's the easier solution lol. I should just do that